### ISSUE TITLE
generalization ability

### ISSUE THREAD

**Issue by brightmart** [siddk/entity-network#3]
it seems that there is no reguarization for the model in this implementation. and it may be lack of generalization ability. 
will this be a issue, or should reguarization to added( e.g. L2 reguarization)?

**Comment by siddk** [comment]
Yeah - I agree. I'm considering adding dropout, and seeing how that affects results.
